app/subscriber/src/components/modal/Modal.tsx

```diff
@@ -44,6 +46,7 @@ export const Modal: React.FC<IModalProps> = ({
   customButtons,
   type,
   hasHeight,
+  enableConfirm,
```

Comment:
This property won't do anything.  How should it be implemented below?

Reply:
It is being used in the ShareMenu.tsx file in the Share Content Modal as: enableConfirm={content.length > 0 && (user?.colleague !== undefined || emailAddress !== '')}

Reply:
Without any logic below, it doesn't make sense to have this property.  Check the Modal in tno-core to see how it was implemented.